src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ChplAPISteps.java

```diff
@@ -24,4 +34,50 @@ public void verifyContentUnderCertifiedHealthITProductListingAndURL(final String
         assertEquals(actualText, text);
         assertEquals(actualURL, urlLink);
     }
+
+    /**
+     * Assert that title should be CHPL API.
+     * @param title expected as CHPL API
+     */
+    @Then("^CHPL API page title should be \"([^\"]*)\"$")
+    public void chplApiPageTitleShouldBe(final String title) {
+        String chplAPITitle = getDriver().getTitle();
+        Assert.assertEquals(chplAPITitle, title);
+    }
+
+    /**
+     * Click all the controller elements link.
+     * @param controllerName as the name of Controller
+     */
+    @When("^I click on \"([^\"]*)\" link$")
+    public void iClickOnLink(final String controllerName) {
+        WebElement link = ChplAPIPage.controllerLink(getDriver(), controllerName);
+        ((JavascriptExecutor) getDriver()).executeScript("arguments[0].click();", link);
+    }
+
+    /**
+     * Assert that APIs from webpage is present in the example API Endpoints.
+     * @param apiEndpoints as the details of controller
+     */
+    @Then("^list of \"(.*)\" operations should be displayed without deprecated calls$")
+    public void listOfOperationsShouldBeDisplayed(final String apiEndpoints) {
+        WebDriver driver = getDriver();
+        // Get element from the webpage into the list
```

Comment:
I don't think you need all of these comments, but if you're going to have them, they ought to be accurate and grammatically correct. In this case it's not `Get element from...` but instead `Get elements from...`, though the `...into the list` is weird. Maybe `Load elements from the page into a list`?

Honestly, and this might just be me, trying to read code where every other line is a comment is jarring. It's like I have to switch back and forth from English to Java constantly, and that's disconcerting